tests/Tests/100__deploy/130__operators/130__rhods_operator/130__rhods_operator.robot

```diff
@@ -35,6 +36,14 @@ Verify RHODS operator information
 
   Run Keyword IF     "mailto:undefined" not in $temp_list     FAIL    There shouldn't be reference to maintainers email
 
+Upgrade And Verify RHODS Via Addon
+    [Documentation]  This TC verfiy if the user is able to upgrade RHODS
+    ...   Via Addon withought monitoring 
+    [Tags]  ODS-1766
+    Upgrade RHODS From Installed Operator
+    ${podStatus}  Get Pod Status    redhat-ods-operator      name=rhods-operator
+    Should Not Be Equal    ${podStatus}   Running
```

Comment:
would it be better to check if the status is something? like "init" or whatever. Because if the status if "failed" in this case the test would last for 5 minutes